src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteConnection.java

```diff
@@ -48,7 +49,7 @@
     private static final int MAXIMUM_BATCH_SIZE = 5000;
     private static final long serialVersionUID = -8227257938510897604L;
     private final MapRouletteConfiguration configuration;
-    private final URIBuilder uriBuilder;
+    private final Supplier<URIBuilder> uriBuilder;
```

Comment:
You need to make URIBuilder class serializable because variables in a serializable class either need to be transient or serializable. That's why you are getting a code smell error

Reply:
I was ignoring that, as that class doesn't really need to be serializable. I can make this transient though.